deps.qt/qt5.ps1

```diff
@@ -103,13 +118,18 @@ function Configure {
         BuildPath = "."
         BuildCommand = "${BuildCommand}"
         Target = $Target
-        HostArchitecture = $Target
+        HostArchitecture = Get-HostArchitecture
```

Comment:
@RytoEX is this correct? It seems wrong to me because that would force the DevShell to use the host architecture compiler (e.g. x64) to build another target (i.e. x86) and would fail.

Reply:
HostArchitecture appears to set the host toolchain rather than specifying the what the host OS architecture. While the dev shell may be able to cross-compile, and will be necessary for arm64 on CI until there are WoA runners, it's best to avoid cross-compiling when possible, and we don't want to force x86 cross-compiles from the x64 host toolchain. In my experience, some projects also don't handle such instances well. They may not build, or they may build, but incorrectly.